### DISCUSSION TITLE
Recent Activity

### DISCUSSION THREAD

**Discussion by spyder007** [Donkie/Spoolman#340] [General]
I noticed a number of pull requests are awaiting approvals and actions.  How active is this project, generally?  I'd certainly be willing to help out in maintaining this one, as I plan on using it pretty extensively.

**Comment by Donkie** [comment]
Hey, yeah I've been a bit burned out lately, mostly due to increased stress in my regular work, so programming has not been as enjoyable in my past time. I will try to go through everything asap, but if I fail to then I will consider adding you as a maintainer since I definitely don't want this project to end up in the grave.

**Reply by spyder007** [reply]
@Donkie I get it: my open source contributions come in waves too, depending on everything from workload at work to the weather (when it is nice out, I go out more!)

Many hands make light work, as they say, so if you ever consider expanding administration of the project, please keep me in mind.  Otherwise, I appreciate the effort and keep up the good work!!
